src/sidebar/components/Annotation/AnnotationReplyToggle.tsx

```diff
@@ -19,18 +19,19 @@ function AnnotationReplyToggle({
   const toggleText = `${toggleAction} (${replyCount})`;
 
   return (
-    <ButtonBase
+    <Button
       classes={classnames(
         // This button has a non-standard color combination: it uses a lighter
-        // text color than other LinkButtons
+        // text color than other Buttons
         'text-grey-7 enabled:hover:text-brand-dark',
-        'no-underline enabled:hover:underline'
```

Comment:
I wonder if this was a `LinkButton` at some point, necessitating the `no-underline` rule here? Dunno.